epochStart/bootstrap/export_test.go

```diff
@@ -268,5 +268,3 @@ func (e *epochStartMetaBlockProcessor) GetMapMetaBlock() map[string]*block.MetaB
 const DurationBetweenChecksForEpochStartMetaBlock = durationBetweenChecks
 
 const DurationBetweenReRequest = durationBetweenReRequests
-
-const MinNumOfPeersToConsiderBlockValid = minNumOfPeersToConsiderBlockValid
```

Comment:
You can remove also the two consts above

Reply:
done